test/integration/spec/localparticipant.js

```diff
@@ -1002,4 +1002,33 @@ describe('LocalParticipant', function() {
       });
     });
   });
+
+  // NOTE (mmalavalli): This test ensures that the behavior described in
+  // https://issues.corp.twilio.com/browse/JSDK-2212 is not observed.
+  describe('JSDK-2212', () => {
+    [
+      ['disabled', 'disable'],
+      ['enabled', 'enable']
```

Comment:
Shouldn't we also test for stopped?

Reply:
@syerrapragada We do not listen to the "stopped" event internally in 1.x.